### ISSUE TITLE
Poor app unlock user interface 

### ISSUE THREAD

**Issue by ArchiveBase** [wireapp/wire-android#2486]
![Screenshot_2019-12-16-16-32-52](https://user-images.githubusercontent.com/46082456/70902768-0dc8bd80-2030-11ea-8f23-c7f77cfe3bd9.png)

**Comment by makingthematrix** [comment]
Right. I see that the background is not visible for the biometric prompt (fixed in https://github.com/wireapp/wire-android/pull/2508/), but it's still visible for the regular dialog.
I created a JIRA ticket: https://wearezeta.atlassian.net/browse/AN-6609
